packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -318,9 +319,7 @@ export class PHPRequestHandler {
 	 * @param  request - PHP Request data.
 	 */
 	async request(request: PHPRequest): Promise<PHPResponse> {
-		const isAbsolute =
-			request.url.startsWith('http://') ||
-			request.url.startsWith('https://');
+		const isAbsolute = isAbsoluteUrl(request.url);
```

Comment:
I moved the absolute url check to a helper function in a previous version of the code where I wanted to reuse the check.

It seems like a nice helper to have around, so I kept it in the PR.